lib/serverless-tools/deployer/ruby_builder.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module ServerlessTools
+  module Deployer
+    class RubyBuilder
+      attr_reader :config
```

Comment:
Are there any places where we access config from the outside? Could it potentially be made private?